webui.dev/js/marshaller.js

```
@@ -158,36 +158,40 @@ function onExecutionFinished (evt) {
     return
   }
 
+  let feedbackButton = actionButton;
+
   switch (actionButton.popupOnStart) {
     case 'execution-button':
-      if (document.querySelector('execution-button#execution-' + logEntry.executionTrackingId) !== null) { // If the button was created in our instance
-        document.querySelector('execution-button#execution-' + logEntry.executionTrackingId).onExecutionFinished(logEntry)
-      }
+	  let executionButton = document.querySelector('execution-button#execution-' + logEntry.executionTrackingId)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)